src/heuristicblocking.js

```diff
@@ -181,9 +204,6 @@ HeuristicBlocker.prototype = {
     if(firstParties.indexOf(page_origin) === -1){
       firstParties.push(page_origin);
       snitch_map.setItem(tracker_origin, firstParties);
-      // TODO removing this line/messing up parameters should break integration tests
-      this.storage.setupHeuristicAction(tracker_fqdn, constants.ALLOW);
-      this.storage.setupHeuristicAction(tracker_origin, constants.ALLOW);
```

Comment:
By the way, this logic was introduced in https://github.com/EFForg/privacybadger/pull/819/commits/e225ad60b88d99b62315efd1e54629648e9ef2af as part of [de-ABPification](https://github.com/EFForg/privacybadger/pull/819).